R/sprite.R

```diff
@@ -0,0 +1,76 @@
+idExists <- function(id, sprite) {
```

Comment:
Let's use snake case everywhere in R code, when we specify new variables, functions etc